### PR TITLE
feat: allow configuration via amagaki.ts

### PR DESCRIPTION
**Pull request by jeremydw**
Based on implementation via https://github.com/blinkk/oak/blob/main/src/cms/pod.ts#L125

Nice feature as it validates the user's pod configuration and doesn't require them to fuss with tsc or similar.

### CODE REVIEW

File: src/pod.ts [hunk 1]
```diff
@@ -56,7 +60,7 @@ export class Pod {
     defaultLocale: 'en',
     locales: ['en'],
   };
-  static DefaultConfigFile = 'amagaki.js';
+  static DefaultConfigFiles = ['amagaki.ts', 'amagaki.js'];
```
Comment (stevenle) [inline src/pod.ts:63]: I think the .js file should take precedence in the case that the user decides to precompile their ts file to js.

Comment (jeremydw) [reply src/pod.ts]: Definitely; thanks for catching that. I had it that way but inadvertently flipped it when I was updating the docs.

**Comment by jeremydw** [comment]
Apologies, I inadvertently merged this when I meant to do so on another repo/project. LMK if you have more feedback and I'll address in a subsequent commit.

**Review by stevenle — COMMENTED** [review]
LGTM

**Pull request merged**